app/views/spree/orders/_totals_footer.html.haml

```diff
@@ -30,14 +30,23 @@
       %td.text-right.total
         %strong
           = order.display_payment_total.to_html
-  - if order.outstanding_balance?
+  - if order.outstanding_balance? && order.outstanding_balance > 0
     %tr.total
       %td.text-right{colspan: "3"}
         %h5.not-paid
           = t :order_balance_due
       %td.text-right.total.not-paid
         %h5.not-paid
           = order.display_outstanding_balance.to_html
+  - else
+    %tr.total
+      %td.text-right{colspan: "3"}
+        %h5
+          = t :credit_owed
+      %td.text-right.total.not-paid
```

Comment:
It sounds like this class `not-paid` is not intentional here. Perhaps the colour is overwritten by the `h5` style, but it would be better to remove this if it's not required.

Reply:
I removed the not-paid styling for the credit owed field.